File: android/src/main/java/expo/modules/langpicker/LangPickerPluginModule.kt
package expo.modules.dylmye

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition
import java.net.URL

class LangPickerPluginModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("LangPickerPlugin")
    AsyncFunction("setValueAsync") { value: String ->
      // Send an event to JavaScript.
      sendEvent("onChange", mapOf(
        "value" to value
      ))
    }
  }
}

File: ios/LangPickerPluginModule.swift
import ExpoModulesCore

public class LangPickerPluginModule: Module {
  public func definition() -> ModuleDefinition {
    Name("LangPickerPlugin")
    AsyncFunction("setValueAsync") { (value: String) in
      // Send an event to JavaScript.
      self.sendEvent("onChange", [
        "value": value
      ])
    }
  }
}
